site-packages/integralstor/update_system_date_time.py

```diff
@@ -0,0 +1,28 @@
+from integralstor_utils import command
+
+def update_system_date_time(date, time):
+    try:
+	if not time == None:
+            cmd = 'date -s %s' %time
+            output, err = command.get_command_output(cmd)
+            if err:
+                raise Exception(err)
+        if not date == None:
```

Comment:
This is a double negative check. Instead you should actually just say, 
if date:
  do something..